src/lib.rs

```diff
@@ -30,8 +30,61 @@ use holochain_zome_types::{
 };
 type HappIds = Vec<String>;
 
-pub async fn activate_holo_hosted_happs(core_happ: Happ) -> Result<()> {
-    let list_of_happs = get_enabled_hosted_happs(core_happ).await?;
+use ed25519_dalek::{PublicKey, SecretKey};
+use hpos_config_core::*;
+
+pub async fn handle_test_network_registration(core_happ: &Happ) -> Result<()> {
+    let hpos_config_path = env::var("HPOS_CONFIG_PATH")?;
+    let is_test_network = env::var("IS_TEST_NETWORK").is_err();
```

Comment:
So `is_test_network` is `true` when `$IS_TEST_NETWORK` is not set? e.g. the intended usecase for `$IS_TEST_NETWORK` is only `IS_TEST_NETWORK=false`?

Reply:
Oops! My mistake cargo culting example code :)